amivapi/tests/events/test_emails.py

```diff
@@ -229,3 +229,18 @@ def test_no_nones_in_emails(self):
         mail = self.app.test_mails[1]
         for field in mail.values():
             self.assertTrue('None' not in field)
+
+
+    def test_moderator_signup_text(self):
+        text = "We will meet at 9:30."
+        event = self.new_object('events', spots=100, selection_strategy='fcfs', moderator_signup_text=text)
+        user = self.new_object('users')
+
+        signup = self.api.post('/eventsignups', data={
+            'user': str(user['_id']),
+            'event': str(event['_id'])
+        }, status_code=201).json
+
+        mail = self.app.test_mails[0]
+
+        self.assertTrue(text not in mail)
```

Comment:
This line only checks whether the mail dict has a key with the message stored in `text`.

It should rather be something like:

```suggestion
        self.assertTrue(text not in mail['text'])
```